config/passport.js

```diff
@@ -51,32 +52,46 @@ module.exports = function(passport, models) {
                 email: profile.emails[0].value,
                 provider: 'google'
             }, defaults: {
-                    email: profile.emails[0].value,
-                    provider: 'google',
-                    google_token: accessToken,
-                    google_id: profile.id,
-                    fname: fname,
-                    lname: lname
-                }})
+                email: profile.emails[0].value,
+                provider: 'google',
+                google_token: accessToken,
+                google_id: profile.id,
+                fname: fname,
+                lname: lname
+            }})
                 .spread((user, created) => {
                     if (!created && !user) {
                         return done(null, false, {info: errorCode.PCE010.message, code: errorCode.PCE010.code});
                     }
                     progress.findOrCreate({where: {
                         user_id: user.id
                     }, defaults: {
-                            user_id: user.id,
-                            stage: 0,
-                            activity: 0
-                        }})
+                        user_id: user.id,
+                        stage: 0,
+                        activity: 0
+                    }})
                         .spread((status, created) => {
                             const response = {email: user.email, name: user.name};
                             return done(null, response);
                         })
                         .catch(function(err) {
                             return done(err);
                         });
-                })
+                    levelProgress.findOrCreate({where: {
+                        user_id: user.id
+                    }, defaults: {
+                        user_id: user.id,
+                    }})
+                        .spread(() => {
+                            const response = {email: user.email, name: user.name};
+                            return done(null, response);
+                        })
+                        .catch(function(err) {
+                            return done(err);
+                        });
+                }
+
```

Comment:
Extra line here also, I think this can be removed.

Reply:
Done.